tests/test_visitors/test_ast/test_builtins/test_numbers/test_approximate_constants.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    ApproximateConstantViolation,
+)
+from wemake_python_styleguide.visitors.ast.builtins import WrongNumberVisitor
+
+
+@pytest.mark.parametrize(('variable_name', 'variable_value'), [
```

Comment:
We also need a test here when violation is not raised. Cases to cover:
1. Value is completely different, like `100`
2. Value is slightly different, like `3.15`

And the same should cover all constants we support.

Reply:
I think we need 0.005 as MATH_CONSTANTS_MAX_DIFF, or 3.15 causes violation

Reply:
Ok!